src/synopsys-action/tools-parameter.ts

```diff
@@ -185,15 +186,16 @@ export class SynopsysToolsParameter {
     }
 
     // Check and put environment variable for fix pull request
-    if (inputs.BLACKDUCK_AUTOMATION_FIXPR.toLowerCase() !== 'false') {
+    if (parseBoolean(inputs.BLACKDUCK_AUTOMATION_FIXPR)) {
       info('Blackduck Automation Fix PR is enabled')
       blackduckData.data.github = this.setGithubData()
+      blackduckData.data.blackduck.automation.fixpr = true
     } else {
       // Disable fix pull request for adapters
       blackduckData.data.blackduck.automation.fixpr = false
     }
 
-    if (inputs.BLACKDUCK_AUTOMATION_PRCOMMENT) {
+    if (parseBoolean(inputs.BLACKDUCK_AUTOMATION_PRCOMMENT)) {
       info('Blackduck Automation comment is enabled')
       blackduckData.data.github = this.setGithubData()
       blackduckData.data.blackduck.automation.prcomment = Boolean(inputs.BLACKDUCK_AUTOMATION_PRCOMMENT)
```

Comment:
if BLACKDUCK_AUTOMATION_PRCOMMENT is not set do we need to pass it as false to bridge or we won't pass it at all like we have for blackduckData.data.blackduck.automation.fixpr

Reply:
done